nncf/common/tensor_statistics/aggregator.py

```diff
@@ -90,10 +113,30 @@ def register_statistic_points(self, statistic_points: StatisticPointsContainer)
             for _statistic_point in _statistic_points:
                 for _, tensor_collectors in _statistic_point.algorithm_to_tensor_collectors.items():
                     for tensor_collector in tensor_collectors:
-                        if self.stat_subset_size is None:
-                            self.stat_subset_size = tensor_collector.num_samples
+                        if self.iterations_number is None:
+                            self.iterations_number = tensor_collector.num_samples
                         elif tensor_collector.num_samples is not None:
-                            self.stat_subset_size = max(self.stat_subset_size, tensor_collector.num_samples)
+                            self.iterations_number = max(self.iterations_number, tensor_collector.num_samples)
+
+    def is_model_has_no_batch_axis(self, graph: NNCFGraph) -> bool:
```

Comment:
I think this function should not be a part of `StatisticsAggregator`.

Reply:
removed. all logs connected to batchwise_statistics is put inside quntize_impl() and its variations